### DISCUSSION TITLE
IBM i debug getting CPD0063  Too many elements in the list for parameter :INLLIBL

### DISCUSSION THREAD

**Discussion by fabiokang** [codefori/vscode-ibmi#1755] [IBM i] [answered]
Hi , One of our developer opens a code and select the debug for i and it is getting CPD0063, Too many elements in the library list for parameter  INLLIBL. We already  changed her job description  to have minimal library list ,but she is still getting the error and can't debug the program.
There is a pop screen with this message and options "Open launch.json" and Cancel

when she clicks on  Open launch.json there is box stating "Select a workspace folder to create a launch.json  file in or add to the workspace config file"

When she clicks "Cancel" we can see the output for IBM will attached.

We are not sure how to fix the CPD0063 problem.

**Comment by fabiokang** [comment]
[CPD0063.txt](https://github.com/codefori/vscode-ibmi/files/13813090/CPD0063.txt)

**Reply by fabiokang** [reply, marked as answer]
I checked the developer  new job- Edit Connection Settings- Library List , we noticed a library name duplicated in the list. We removed  the duplicated entry and it works now.

**Comment by fabiokang** [comment]
a single library name was twice in the Edit Connection Settings - Library List

**Reply by chrjorgensen** [reply]
If you have issues where you suspect an invalid library list, you can have Code for IBM i validating your library list and remove any invalid libraries from the list:

![image](https://github.com/codefori/vscode-ibmi/assets/13275072/5ae95327-f36c-42ae-b200-0076e44a9d22)

You will also find the command in F1 -> "clean"...
